docs/source/explore/gallery.rst

```diff
@@ -0,0 +1,60 @@
+#######
+Gallery
+#######
+
+Here are some pictures of past events and Interface elements.
+
+
+.. figure:: _images/gallery/el_papa_viejo.jpg
+    :alt: Maker meeting 28th September 2020
+    :class: inline
+
+    Maker meeting, 28th September 2020, on the "El Papa Viejo" (meaning "The Big Pope").
```

Comment:
```suggestion
    Maker Meeting, 28th September 2020, on the "El Papa Viejo" (meaning "The Old Pope").
```

Reply:
Viejo means "old" instead of "big" from as far as I can find from internet resources.